frontend/src/components/FormCreateCause.jsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from "react";
+import "./styles/FormCreateCause.scss";
+import axios from "axios";
+import CheckboxList from "../CheckboxList";
+const { apiCall } = require("../conf");
+
+export default function FormCreateCause() {
+  const [asso, setAsso] = useState({
+    name: "",
+    img: null,
+    resume: "",
+    website: "",
+    id_tag: []
+  });
+
+  const data = asso;
+
+  const handleSubmit = () => {
+    axios
+      .post(`${apiCall}/association`, data)
+      .then(res => {
+        alert(res.data);
```

Comment:
Don't you think there are better ways to inform your visitors ? Like, a toast ?
(`react-toastify` is very easy to use and very nice to the eye !)